samples/samples/autocommit.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+import argparse
+
+from google.cloud.spanner_dbapi import connect
+
+
+def enable_autocommit_mode(instance_id, database_id):
+    """Enables autocommit mode."""
+    # [START enable_autocommit_mode]
+    connection = connect(instance_id, database_id)
+    connection.autocommit = True
+    print("Autocommit mode is enabled.")
+
+    cursor = connection.cursor()
+
+    cursor.execute(
+        """CREATE TABLE Singers (
+                SingerId     INT64 NOT NULL,
+                FirstName    STRING(1024),
+                LastName     STRING(1024),
+                SingerInfo   BYTES(MAX)
+            ) PRIMARY KEY (SingerId)"""
+    )
```

Comment:
We probably could move data insertion into the test to shorten the sample, but that is something to ask googlers.